### DISCUSSION TITLE
Develop the Project

### DISCUSSION THREAD

**Discussion by Paramjeet2901** [community/community#125508] [Other Feature Feedback, Questions, & Ideas]

**Comment by ebndev** [comment]
Hi @Paramjeet2901, we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
